### PR TITLE
added health-r14-swagger

### PR DESCRIPTION
**Pull request by sebastianschramm**
simple backend (tornado) stub with swagger api for customer workshop next week

### CODE REVIEW

File: templates/health-r14-swagger/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+services:
+  r14swagger:
+    image: ${r14_swagger_image}
+    labels:
+      io.rancher.scheduler.affinity:container_label_soft_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
+      io.rancher.container.pull_image: always
+      io.rancher.container.hostname_override: container_name
+
+  lb:
```
Comment (flypenguin) [inline templates/health-r14-swagger/0/docker-compose.yml]: you only have one container - why the load balancer?

File: templates/health-r14-swagger/0/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+version: '2'
+
+services:
+  r14swagger:
+    image: ${r14_swagger_image}
```
Comment (flypenguin) [inline templates/health-r14-swagger/0/docker-compose.yml]: I would make this part static: `hub.cognotekt.int:5000/cvs/r14` - so you only have to enter the tag.

Comment (sebastianschramm) [inline templates/health-r14-swagger/0/docker-compose.yml]: ok, done

File: templates/health-r14-swagger/0/docker-compose.yml (hunk 1 above)
Comment (sebastianschramm) [inline templates/health-r14-swagger/0/docker-compose.yml]: removed it

**Comment by sebastianschramm** [comment]
@flypenguin can you please have a look at my changes? thank you :1st_place_medal:

File: templates/health-r14-swagger/0/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      description: "Choose 'webui_external' for reachability from 'outside'"
+      type: enum
+      options:
+        - webui
+        - webui_external
+      required: true
+
+services:
+  r14swagger:
+    scale: 1
```
Comment (flypenguin) [inline templates/health-r14-swagger/0/rancher-compose.yml]: you could do two. you can also scale by hand later, if you need to. just fyi.

**Pull request merged**